traitcuration/traits/templates/traits/base.html

```diff
@@ -35,11 +37,23 @@
             {% endwith %}
         </div>
         <div class="uk-navbar-right">
-            <ul class="uk-navbar-nav">
-                <li class="header__item">
-                    <a class="header__link" href="#" uk-icon="icon: user"></a>
-                </li>
-            </ul>
+            {% if user.is_anonymous %}
+                <button class="uk-button button button-outlined--transparent">
+                    <a href="/accounts/google/login/?next={{request.path}}">
+                        <span uk-icon="google" class="uk-margin-small-right"> </span> LOG IN 
+                    </a> 
+                </button>
+            {% else %}
+                <div class="auth-links">
+                    <span uk-icon="icon: user; ratio: 1.3" class="auth-links__icon"></span>
+                    <span>{% user_display user %}</span>
+                    <button class="uk-button button button-outlined--transparent">
+                        <a href="/accounts/logout/?next={{request.path}}">
+                            <span uk-icon="sign-out" class="uk-margin-small-right"> </span> SIGN OUT 
```

Comment:
When logged in, the email is too close to the "Sign out" button:

![image](https://user-images.githubusercontent.com/10669118/87936687-2b10af00-ca8b-11ea-8d54-b0c2b3cb31c2.png)

Reply:
Same problem with PR #63 , looks like the `gap` CSS property isn't fully supported. May I ask what browser are you using?

Reply:
Looks like you fixed it, since now it looks good:
![image](https://user-images.githubusercontent.com/10669118/88379769-636e0100-cd9b-11ea-901e-4bac3725f8ce.png)

I'm always using the latest(ish) release Google Chrome on Ubuntu. My current version is: 83.0.4103.97 (Official Build) (64-bit)